### PR TITLE
DOC: fixed labels in "Plotting with error bars"

### PR DESCRIPTION
**Pull request by marydmit**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

x-labels should now be visible in the plot.

### CODE REVIEW

**Review by MarcoGorelli — APPROVED** [review]
Looks much better, thanks @marydmit !

File: doc/source/user_guide/visualization.rst [hunk 1]
```diff
@@ -1436,7 +1436,7 @@ Here is an example of one way to easily plot group means with standard deviation
 Plotting tables
 ~~~~~~~~~~~~~~~
 
-Plotting with matplotlib table is now supported in  :meth:`DataFrame.plot` and :meth:`Series.plot` with a ``table`` keyword. The ``table`` keyword can accept ``bool``, :class:`DataFrame` or :class:`Series`. The simple way to draw a table is to specify ``table=True``. Data will be transposed to meet matplotlib's default layout.
+Plotting with matplotlib table is now supported in  :meth:`DataFrame.plot` and :meth:`Series.plot` with a ``table`` keyword. The ``table`` keyword can accept ``bool``, :class:`DataFrame` or :class:`Series`. The simple way to draw a table is to specify ``table=True``. Data will be transposed to meet matplotlib's default layout.
```
Comment (jorisvandenbossche) [inline doc/source/user_guide/visualization.rst:1439]: Can you remove the added whitespace?
```suggestion
Plotting with matplotlib table is now supported in  :meth:`DataFrame.plot` and :meth:`Series.plot` with a ``table`` keyword. The ``table`` keyword can accept ``bool``, :class:`DataFrame` or :class:`Series`. The simple way to draw a table is to specify ``table=True``. Data will be transposed to meet matplotlib's default layout.
```

**Review by jorisvandenbossche — COMMENTED** [review]
Small comment, for the rest looks all good. Thanks for the contribution!

File: doc/source/user_guide/visualization.rst (hunk 1 above)
Comment (charlesdong1991) [inline doc/source/user_guide/visualization.rst:1439]: a trailing space seems to be accidentally added here

**Review by jorisvandenbossche — APPROVED** [review]

File: doc/source/user_guide/visualization.rst [hunk 2]
```diff
@@ -1445,9 +1445,9 @@ Plotting with matplotlib table is now supported in  :meth:`DataFrame.plot` and :
 
 .. ipython:: python
 
-   fig, ax = plt.subplots(1, 1)
+   fig, ax = plt.subplots(1, 1, figsize=(7,6.5))
```
Comment (jorisvandenbossche) [inline doc/source/user_guide/visualization.rst:1448]:
```suggestion
   fig, ax = plt.subplots(1, 1, figsize=(7, 6.5))
```

There are some linting errors (like the above):

```
##[error]doc/source/user_guide/visualization.rst:1447:43:E231:missing whitespace after ','
##[error]doc/source/user_guide/visualization.rst:1449:23:E261:at least two spaces before inline comment
##[error]doc/source/user_guide/visualization.rst:1466:43:E231:missing whitespace after ','
##[error]doc/source/user_guide/visualization.rst:1467:23:E261:at least two spaces before inline comment
```

**Review by jorisvandenbossche — CHANGES_REQUESTED** [review]

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**